### PR TITLE
Add GitHub CI

### PR DESCRIPTION
**Pull request by madebr**
Over the past weekend, I added support for SDL satallite libraries to [libsdl-org/setup-sdl](https://github.com/libsdl-org/setup-sdl).
(it currently only supports SDL3)

This seems like a good opportunity to add ci to SDL_rtf.

https://github.com/madebr/SDL_rtf/actions/runs/7532633083

### CODE REVIEW

File: examples/showrtf.c [hunk 1]
```diff
@@ -99,7 +99,7 @@ static void * SDLCALL CreateFont(const char *name, RTF_FontFamily family, int ch
             TTF_style |= TTF_STYLE_ITALIC;
         if ( style & RTF_FontUnderline )
             TTF_style |= TTF_STYLE_UNDERLINE;
-        TTF_SetFontStyle(font, style);
+        TTF_SetFontStyle(font, TTF_style);
```
Comment (sezero) [inline examples/showrtf.c:102]: This should go into SDL2 branch too (and even to SDL-1.2 branch, as far as I can see..)

File: src/SDL_rtf.c [hunk 2]
```diff
@@ -121,6 +121,10 @@ int RTF_Load_RW(RTF_Context *ctx, SDL_RWops *src, int freesrc)
             RTF_SetError("Couldn't find font for text");
             retval = -1;
             break;
+        default:
+            RTF_SetError("Unknown error");
+            retval = -1;
```
Comment (sezero) [inline src/SDL_rtf.c:126]: This should go into SDL2 branch too

**Review by sezero — APPROVED** [review]

File: src/SDL_rtf.c (hunk 2 above)
Comment (madebr) [reply src/SDL_rtf.c:126]: 80d8cd0f6bc360bdf90c2704c1533257f31cfb8c

File: examples/showrtf.c (hunk 1 above)
Comment (madebr) [reply examples/showrtf.c:102]: c4492f95f472d25cdc7783267692664c1f618cbc

**Comment by madebr** [comment]
Looks like actions are not activated in this repo.

**Comment by sezero** [comment]
Maybe @slouken should activate it??

**Comment by slouken** [comment]
Done!

**Pull request merged**